components/molecules/TableHeader/table-header.tsx

```diff
@@ -95,12 +95,7 @@ const TableHeader = ({
         />
       </div>
       <div className="flex flex-col-reverse items-start gap-3 md:flex-row md:items-end ">
-        {range ? (
-          <ComponentDateFilter setRangeFilter={(range: number) => setRangeFilter?.(range)} defaultRange={range} />
-        ) : (
-          ""
-        )}
-
+        {setRangeFilter && range ? <ComponentDateFilter setRangeFilter={setRangeFilter} defaultRange={range} /> : null}
```

Comment:
Only display the range filter in the header now if the `setRangeFilter` and `range` props are passed in.